app/views/attachments/_attachment_box.html.erb

```diff
@@ -62,30 +62,11 @@
       </tbody>
     </table>
 
-    <div id="drop-zone" class="drop-zone" dropzone="" data-toggle="tooltip" data-placement="right" title="Drag attachments from your desktop and drop them here" ondrop="">Drop zone</div>
+    <div id="drop-zone" class="drop-zone" dropzone="" data-toggle="tooltip" data-placement="right" title="Drag attachments from your desktop and drop them here" ondrop="" data-behavior="drop-zone">Drop zone</div>
 
     <!-- The file upload form used as target for the file upload widget -->
     <%= form_tag project_node_attachments_path(node.project, node, format: :json), id: 'fileupload', multipart: true do %>
-
-      <!-- The fileupload-buttonbar contains buttons to add/delete files and start/cancel the upload -->
-      <div class="fileupload-buttonbar">
- 
-        <!-- The fileinput-button span is used to style the file input field as button -->
-        <span class="btn btn-primary fileinput-button" data-toggle="tooltip" data-placement="bottom" title="Add...">
-          <i class="fa fa-plus fa-inverse"></i>
-          <span></span>
-          <input type="file" name="files[]" multiple>
-        </span>
-        <button type="submit" class="btn btn-success start" data-toggle="tooltip" data-placement="bottom" title="Start">
-          <i class="fa fa-upload fa-inverse"></i>
-          <span></span>
-        </button>
-        <button type="reset" class="btn btn-warning cancel" data-toggle="tooltip" data-placement="bottom" title="Cancel">
-          <i class="fa fa-ban fa-inverse"></i>
-          <span></span>
-        </button>
-
-      </div>
```

Comment:
Why was the upload button removed?

Reply:
We are auto-uploading files. As soon as a user selects a file via upload button, drops onto a drop zone or pastes into a text area, the file(s) are uploaded right away rather than being staged like they were previously. This is to facilitate a more seamless user experience when users drag/drop or paste into text areas (we want the files to be available right away not staged in the sidebar).

The buttons that should have been removed were just the `start` and `cancel` buttons, along with the buttons that should up in the staged queue. The `add` button should remain to give users the option to upload by browsing their file system.

Reply:
I re-added the `add` button in the sidebar and removed the `start` button from the staged queue